core/modules/filter/js/filter.admin.js

```diff
@@ -329,11 +329,11 @@ Backdrop.FilterStatus.prototype.addHTMLRule = function (rule) {
  *     restrictions are applied.
  *  2. all nested within the "properties" key: use the "tags" subkey to list
  *     HTML tags to which you want to apply property restrictions, then use the
- *     "allowed" subkey to whitelist specific property values, and similarly use
- *     the "forbidden" subkey to blacklist specific property values.
+ *     "allowed" subkey to allowlist specific property values, and similarly use
+ *     the "forbidden" subkey to denylist specific property values.
```

Comment:
Same as previous comment/suggestion:

```suggestion
 *     "allowed" subkey to allow specific property values, and similarly use the
 *     "forbidden" subkey to deny specific property values.
```

Reply:
...or alternatively:

```suggestion
 *     "allowed" subkey to accept specific property values, and similarly use
 *     the "forbidden" subkey to reject specific property values.
```